kitsune/products/api_utils.py

```diff
@@ -0,0 +1,22 @@
+from django.core.exceptions import ValidationError
+
+from rest_framework import serializers
+
+
+class TopicField(serializers.SlugRelatedField):
```

Comment:
This was moved from kitsune/products/api.py. As you can see, it also got a lot simpler thanks to changes in DRF's serializers.